src/text/isMatch.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Test a string against regexp
+ * @param {string} target
+ * @param {string} regExp
+ * @return Boolean
+ */
+
+export const isMatch = (target: string, rex: string | RegExp): Boolean => {
+  return new RegExp(rex).exec(target) !== null;
```

Comment:
In case that `rex` is of type `RegExp`, what happens when calling the constructor?
In src/text/replaceRegExp.ts you first ask for the `typeof` of the variable and then call the constructor for strings only...

Reply:
Good suggest, it was done.